### DISCUSSION TITLE
How to align left File column?

### DISCUSSION THREAD

**Discussion by beetrootpaul** [RafaelGB/obsidian-db-folder#458] [Q&A] [answered]
As in a title. Right now I see my note titles/links aligned center in the File column, but would prefer them aligned left

<img width="801" alt="screenshot 2022-10-01 at 23 46 24" src="https://user-images.githubusercontent.com/92947323/193429489-1fb70de2-7d69-40fd-93ba-3867574afbb6.png">

**Comment by RafaelGB** [comment, marked as answer]
Currently the position configuration are just available for formulas, number and text column

With the time we can add them to File, tags & select types
